inc/modules/export/epub/class-epub.php

```diff
@@ -2711,29 +2681,46 @@ static function hasDependencies() {
 	}
 
 	/**
-	 * Override load template function
-	 * Switch path from /epub201 to /epub3 when possible.
+	 * Write a file to the temporary EPUB directory.
 	 *
-	 * @param string $path
-	 * @param array $vars (optional)
+	 * @param string $filename
+	 * @param array|string $data
+	 * @param array $options (optional)
 	 *
-	 * @return string
+	 * @return void
 	 * @throws \Exception
 	 */
-	protected function loadTemplate( $path, array $vars = [] ) {
+	protected function createEpubFile( string $filename, $data, array $options = [] ): void {
+		$directory = $options['directory'] ?? $this->epubDir;
+		$template = $options['template'] ?? 'html';
 
-		$search = '/templates/epub201/';
-		$replace = '/templates/epub3/';
+		$path = "{$directory}/{$filename}";
 
-		$pos = strpos( $path, $search );
-		if ( false !== $pos ) {
-			$new_path = substr_replace( $path, $replace, $pos, strlen( $search ) );
-			if ( file_exists( $new_path ) ) {
-				$path = $new_path;
-			}
+		if ( ! is_array( $data ) ) {
+			\Pressbooks\Utility\put_contents( $path, $data );
+
+			return;
 		}
 
-		return parent::loadTemplate( $path, $vars );
```

Comment:
The parent `loadTemplate` method still exists, I believe that it could be removed as well. It seems to be used only for EPUBs